iocage/lib/Jail.py

```diff
@@ -991,29 +991,32 @@ def update_jail_state(self) -> None:
 
     def _teardown_mounts(self) -> None:
 
-        mountpoints = list(map(
-            self._get_absolute_path_from_jail_asset,
-            [
-                "/dev/fd",
-                "/dev",
-                "/proc",
-                "/root/compat/linux/proc"
-            ]
+        mountpoints = list(filter(
+            os.path.isdir,
+            map(
+                self._get_absolute_path_from_jail_asset,
+                [
+                    "/usr/bin",
+                    "/dev/fd",
+                    "/dev",
+                    "/proc",
+                    "/root/compat/linux/proc"
+                ]
+            )
         ))
 
-        mountpoints += list(map(
-            lambda x: x["destination"],
-            list(self.fstab)
-        ))
+        iocage.lib.helpers.umount(
+            mountpoints,
+            force=True,
+            logger=self.logger,
+            ignore_error=True
+        )
 
-        for mountpoint in mountpoints:
-            if os.path.isdir(str(mountpoint)):
-                iocage.lib.helpers.umount(
-                    mountpoint,
-                    force=True,
-                    logger=self.logger,
-                    ignore_error=True  # maybe it was not mounted
-                )
+        iocage.lib.helpers.umount(
+            ["-A", "-F", self.fstab.path, "-t", "nullfs"],
```

Comment:
can you please explain the options?

Reply:
`-A` unmounts all entries from fstab
`-F` sets the provided path as fstab
`-t nullfs` ensures we only unmount nullfs mounts

Reply:
what about potential entries of other types?

Reply:
Finally you pointed me to an issue. It's important not to use `-A`, but `-a`. The latter removes the entries from fstab - `-A` removes all from the system :trollface: